data-structures/arrays/python/array_2D_for_loop.py

```diff
@@ -0,0 +1,36 @@
+# Raspberry Pi Foundation
+# Developed as part of Ada Computer Science
+# 
+# Usage licensed under CC BY-NC-SA 4.0
+
+
+def display_all_words(spelling_words):
+    """Display the words of each level in the 2D list"""
+        
+    # Repeat for each level
+    for i in range(0, len(spelling_words)):
```

Comment:
I would get the length of the array on a separate line. I don't think we should put the 0 in as start of range?????? we can just add a note above the code block to remind them how range works???

Reply:
actually - you don't have the 0 in the previous 1d example. Definitely better without!